lib/main_micro_alp.pm

```diff
@@ -179,7 +179,7 @@ sub load_common_tests {
     # Ansible test needs Packagehub in SLE and it can't be enabled in SLEM
     loadtest 'console/ansible' unless (is_staging || is_sle_micro || is_leap_micro || is_alp);
     loadtest 'console/kubeadm' if (check_var('SYSTEM_ROLE', 'kubeadm'));
-    loadtest 'console/year_2038_detection';
+    loadtest 'console/year_2038_detection' unless is_s390x;
```

Comment:
so it should not run on any s390x? or only in zVM? If later, please use `is_zvm` instead of `is_s390x`

Reply:
Thanks @mloviska, given the fact that rollback can't work on s390x platform, I will skip the tests on s390x platform.
even I used some workarounds there, the issue mentioned in poo#127343 can still be seen sporadically.